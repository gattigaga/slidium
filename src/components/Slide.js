import React from "react";
import PropTypes from "prop-types";
import styled from "styled-components";

const Container = styled.svg`
  width: 640px;
  height: 480px;
  background: white;
  border: 1px solid #ccc;
  box-shadow: 0px 0px 32px rgba(0, 0, 0, 0.2);
  margin: auto;
`;

const Slide = () => <Container />;

Slide.propTypes = {};

export default Slide;
