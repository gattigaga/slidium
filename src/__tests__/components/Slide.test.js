import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import Slide from "components/Slide";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Slide />);
});

describe("Slide", () => {
  it("renders correctly", () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
