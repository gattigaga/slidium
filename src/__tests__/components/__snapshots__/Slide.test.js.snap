// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide renders correctly 1`] = `<styled.svg />`;
